boards/posix/nrf52_bsim/argparse.c

```diff
@@ -110,6 +115,16 @@ void nrfbsim_register_args(void)
 		"nosim", "", 'b',
 		(void *)&nosim, cmd_nosim_found,
 		"(debug feature) Do not connect to the phy"},
+		{ false, false, true,
+		"delay_init", "", 'b',
+		(void *)&arg.delay_init, NULL,
+		"If start_offset is used, postpone initialization and startup "
+		"until start_offset is reached (by default not set)"
+		},
+		{ false, false, true,
+		"no_delay_init", "", 'b',
+		NULL, cmd_no_delay_init_found,
+		"Clear delay_init. Note that by default delay_init is not set"},
```

Comment:
Similarly, if we have `delay_init`, why do we have a `no_delay_init`? Since `delay_init` is optional, wouldn't the lack of the `delay_init` indicate the same as `no_delay_init` being set?

Reply:
--no_delay_init is just a nicety to allow users to override a previously given --delay_init  (say if for example the parameters are given from an environment variable), setting --no_delay_init just toggles back to false the delay_init variable. There is no cost of giving people this command line toggle

Reply:
Right - I should have read the commit message more carefully...